packages/dev/serializers/src/glTF/2.0/glTFAnimation.ts

```diff
@@ -836,56 +832,52 @@ export class _GLTFAnimation {
         convertToRightHandedSystem: boolean,
         useQuaternion: boolean
     ) {
-        const animationType = animation.dataType;
         let cacheValue: Vector3 | Quaternion | number;
         inputs.push(time);
 
-        if (value) {

```

Comment:
The issue is actually just this line. `value` can be zero which will cause this code to not push an output value, but it doesn't make sense that we even need to check this since `value` cannot actually be null.